tasks/build-versions-in-details-page.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Build the versions comparison page
+ */
+const gulp = require('gulp');
+const hbs = require('handlebars');
+const gulpHandlebars = require('gulp-handlebars-html')(hbs);
+const fs = require('fs');
+const rename = require('gulp-rename');
+const revReplace = require('gulp-rev-replace');
+const _ = require('lodash');
+const moment = require ('moment');
+const coloredDomains = require('./common/colored-domains.js');
+
+const majorVersion = 'serenity';
+
+module.exports = {generateVersionsInDetailsPage};
+
+gulp.task('build-versions-in-details-page', ['clean-dist','less'], function() {
+    const fileDirectoryDestination = './dist/pim/serenity';
+
+    return generateVersionsInDetailsPage(fileDirectoryDestination);
+});
+
+function generateVersionsInDetailsPage(fileDirectoryDestination) {
+    var versions = JSON.parse(fs.readFileSync('content/versions-in-details/versions.json'));
+
+    _.each(versions, function(version){
+        var releaseDate = moment.utc(version.releaseDate);
+        var supportEndDate = moment.utc(version.supportEndDate);
+        version.releaseDateShortLabel = releaseDate.format('ll');
+        version.releaseDateLongLabel = releaseDate.format('LL');
+        version.supportEndDateShortLabel = supportEndDate.format('ll');
+        version.supportEndDateLongLabel = supportEndDate.format('LL');
+        var timeLeftBeforeSupportEndInMonths = supportEndDate.diff(moment(), 'months');
+        version.timeLeftBeforeSupportEnd = (timeLeftBeforeSupportEndInMonths > 1) ? timeLeftBeforeSupportEndInMonths + ' months' : supportEndDate.diff(moment(), 'days') + ' days';
```

Comment:
and what if it's <0 ?

Reply:
If it's < 0, it's not shown in the interface as it means that the version is not supported anymore. Do you think that still, I need to handle this case?

Reply:
If it never occurs, no need to handle it :)